acceptance/Rakefile

```diff
@@ -70,3 +71,48 @@ task :acceptance do
   beaker += " #{opts}" if opts != ''
   sh beaker
 end
+
+namespace :ci do
+  namespace :test do
+
+    desc 'This task is used by CI to test the package pe-client-tools'
+    rototiller_task :package => [:set_test_target]do |t|
+
+      flags = [
+          {:name => '--xml', :is_boolean => true},
+          {:name => '--no-color', :is_boolean => true},
+          {:name => '--debug', :is_boolean => true},
+          {:name => '--hosts', :override_env => 'TEST_TARGET'},
+          {:name => '--helper', :default => 'helper.rb'},
+          {:name => '--keyfile', :default => "#{ENV['HOME']}/.ssh/id_rsa-acceptance", :override_env => 'BEAKER_KEYFILE'},
+          {:name => '--load-path', :default => 'lib', :override_env => 'BEAKER_LOADPATH'},
+          {:name => '--pre-suite', :default => 'setup/pe/pre_suite', :override_env => 'BEAKER_PRESUITE'},
+          {:name => '--tests', :default => 'tests/', :override_env => 'BEAKER_TESTSUITE'},
```

Comment:
`tests/` is not the proper default for pdb-cli. As written in https://github.com/puppetlabs/puppetdb-cli/pull/53 the default would key off of the `BEAKER_TYPE` variable and be "#{ENV['BEAKER_TYPE']}/tests/".
I am not sure if we want to track the `BEAKER_TYPE` value or just default to `pe/tests`.